server/index.js

```diff
@@ -4,9 +4,14 @@ const compression = require('compression')
 const fs = require('fs')
 const https = require('https')
 const {frontendConfig, backendConfig} = require('./helpers/loadConfig')
+const removeChainLibsWasmImport = require('./helpers/removeChainLibsWasmImport')
 
 let app = express()
 
+express.static.mime.types.wasm = 'application/wasm'
+
+removeChainLibsWasmImport()
```

Comment:
this is not good. it will probably not work in non-devel mode. Instead, we should just run `node helpers/remove...` as a part of `package.json` scripts for running the server

Reply:
agreed, I included it in the `build` script.